packages/astro/src/core/build/index.ts

```diff
@@ -229,6 +229,14 @@ class AstroBuilder {
 				);
 			}
 		}
+
+		if (config.build.split === true) {
+			if (config.output !== 'server') {
+				throw new Error(
+					'The option `build.split` can be used only when `output` is set to `"server"`.'
```

Comment:
```suggestion
					'The option `build.split` can only be used when `output` is set to `"server"`.'
```

I think the `only` in this spot flows a bit natural 🤔